app/models/project.rb

```diff
@@ -28,7 +28,7 @@ def github_repo_url
   end
 
   def git_remote
-    stage = stages&.detect { |s| s.name == 'master' }
+    stage = stages&.detect { |main_stage| %w[main master].any? main_stage.name }
```

Comment:
I didn't know about this style of `any?`!

Reply:
I honestly only prefer it because I am getting `include?` 50% of the time wrong with adding an `s` for the third person flexion. 🙈